### ISSUE TITLE
Restore backup fails on v0.10.0

### ISSUE THREAD

**Issue by r-hmn** [nextcloud/all-in-one#481]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. My existing AIO instance v0.10.0 backs up to /mnt/mybackup
2. My new AIO v.10.0 instance is on another machine. I run container as per https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
3. on https://localhost:8080/containers you can either fill in the domain name, or try to restore a backup.
4. Copy the original /mnt/mybackup contents to the new machine's folder using tar and sudo, so the files are all root:root same as the original system had it.
5. on the page https://localhost:8080/containers, enter /mnt/mybackup  into the path-box, and enter the key which was mentioned during the last backup.
6. press the test button

### Expected behaviour 
success

### Actual behaviour 
Test failed. the log files says (mentions a repsitory which i haven't seen anywhere)
> No 'borg' directory in the given backup directory found!
Please adjust the directory so that the borg archive is positioned in a folder named 'borg' inside the given directory!
Repository /mnt/borgbackup/borg does not exist.

### Tempary fix:
Change my actual folder structure according to the folder mentioned in the error log.
So i changed /mnt/mybackup/   to  /mnt/borgbackup
and i finished successfully

### Host OS 
Lubuntu 21.10

#### Nextcloud AIO version 
v0.10.0 Beta

**Comment by szaimen** [comment]
Hi, I honestly fail to understand where the bug lies. 

It is explained in the interface that the repo folder must be located in a subfolder named borg:
https://github.com/nextcloud/all-in-one/blob/18b3c76a67d79cc1928ed2df503636a11ec26315/php/templates/containers.twig#L145

**Comment by r-hmn** [comment]
The HTML page didn't show the info,.
It was the log file which mentions where it searched and couldn't find it.

**Comment by r-hmn** [comment]
Sorry, i mean. The HTML page isn't the problem :)
The log file mentions where it searched, but it was somewhere else than i entered in.
Once i changed my folder sctructure to be the same as what the log file mentioned, then the restore-function worked

**Comment by szaimen** [comment]
> The log file mentions where it searched, but it was somewhere else than i entered in.

Yes, this is expected because the path that you enter gets mounted somewhere else inside the container and thus not a bug.

> Once i changed my folder sctructure to be the same as what the log file mentioned, then the restore-function worked

I doubt that this was the only way to make this work. As mentioned it only looks for a borg folder in the directory that you enter.

**Comment by r-hmn** [comment]
Strange indeed.
on my original system, i had it set to write to  /mnt/mybackup
on the new system, i created the same folder structure under /mnt/mybackup
using the freshly run mastercontainer, i pointed to '/mnt/mybackup' and entered the code, 
then pressed the "test" buttion. it says it failed, and i inspected the log file.
after checking the ownership of the folders and pressing many times, and pondering..
i tried structuring the actual folder under /mnt to be the same as the (strange)folder what was mentioned in the log-file.
then it worked.  
Could it be that this 'strange folder'( /mnt/borgbackup/ ) was one of the very first stries i did, and that this string is stored somewhere in the backup itself?

**Comment by szaimen** [comment]
> Could it be that this 'strange folder'( /mnt/borgbackup/ ) was one of the very first stries i did, and that this string is stored somewhere in the backup itself?

No. This strange folder is the mountpoint of the directory that you enter inside the container. So it is expected that this string appears in the logs.

**Comment by r-hmn** [comment]
in that case, then i should see that mapping on the container itself as this?    
/mnt/mybackup : /mnt/borgbackup

**Comment by szaimen** [comment]
> ok. in that case, then i should see that mapping on the container itself as this?
> /mnt/mybackup : /mnt/borgbackup
> or is this mounting done in another way

This is how it should work, yes. You can inspect this with `sudo docker inspect nextcloud-aio-borgbackup`

**Comment by r-hmn** [comment]
i ran your command, and at this moment it contains this part (as expected, because after initially trying /mnt/mybackup, i restructured the physical folder as  /mnt/borgbackup and also put that folder in the textbox)
```
 },
            {
                "Type": "bind",
                "Source": "/mnt/borgbackup",
                "Destination": "/mnt/borgbackup",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
```

**Comment by szaimen** [comment]
> Tonight i'll do startup another VM and try another restore attempt as i did. trying to reproduce the same problem and then
> i'll keep an eye on this bind, and see what it does.

That sounds good! Thanks for investigating!

**Comment by r-hmn** [comment]
i tested restoring again. Starting with a clean linuxmint-20.3_xfce VM with docker and portianer
This time, the path /mnt/mybackup was picked up exactly as expected.🤷‍♀️
i both tested by running
```
sudo docker run -it \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest
```

and
```
sudo docker run -it \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 8080:8080 \
-e APACHE_PORT=11000 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest

```

**Comment by szaimen** [comment]
Strange that you werent able to reproduce but cool that it seems to work :)
